Test/e2e/api/subscribe-profile.spec.js

```diff
@@ -0,0 +1,458 @@
+const { test, expect } = require('@playwright/test');
+const { backOff } = require('exponential-backoff');
+const Admin = require('../locators/admin');
+const Storefront = require('../locators/storefront');
+const { createProfileInKlaviyo, checkProfileInKlaviyo, checkProfileListRelationships } = require('../utils/klaviyo-api');
+
+test.describe.configure({ mode: 'serial' }); // This is necessary to ensure that the Klaviyo newsletter config is updated before running the tests
+
+/**
+ * Tests the Klaviyo profile subscription functionality when honor Klaviyo consent is enabled
+ *
+ * Tests the following Klaviyo API functionality:
+ * - Profile creation with SUBSCRIBED consent via footer form
+ * - Profile creation with SUBSCRIBED consent via account creation
+ * - Profile consent updates via newsletter management
+ *
+ * @see https://developers.klaviyo.com/en/reference/bulk_subscribe_profiles
+ */
+test.describe('Profile Subscription - honor Klaviyo consent', () => {
+    test.describe.configure({ mode: 'default' });
+    test.beforeEach(async ({ page }) => {
```

Comment:
Are the tests in each group isolated and can all tests within the group be run in parallel? Could we just update the configuration setting once before running each group of tests i.e. set Profile Subscription to honor Klaviyo consent then run the tests in this group?

Reply:
I tried using a beforeAll but was having issues - so i went w/ beforeEach but the setting doesn't actually get updated if a previous run updated it. unfortunately there isn't a way to make the group of tests run in parallel while the two sets run serially